docker-compose.yaml

```diff
@@ -0,0 +1,24 @@
+version: '3.8'
+
+services:
+  redis:
+    image: redis:latest
+    container_name: redis
+    ports:
+      - "6379:6379"
+    environment:
+      - REDIS_USER=redis-user
+      - REDIS_PASSWORD=redis-password
+    networks:
+      - otp-network
```

Comment:
No need for Redis container.
Check: https://discord.com/channels/1217004414359965798/1264584283360727050/1264671226895536240